scripts/secureboot-certs

```diff
@@ -795,7 +802,14 @@ be passed to {} as custom auth files.
         help="The output file name.",
     )
 
-    if len(sys.argv) == 2 and sys.argv[1] == Actions.INSTALL:
+    if sys.argv[1] == Actions.INSTALL and (
+        len(sys.argv) == 2 or (
+            len(sys.argv) == 3 and (
+                sys.argv[2].startswith("-ua") or
+                sys.argv[2].startswith("--user-agent")
```

Comment:
No sure this will work with `secureboot-certs install --user-agent "My user agent" (len will be 4 here, won't it?).

BTW I'm just realizing right now, I'm not sure the existing code was the cleanest way to support `secureboot-certs install` without arguments. And it becomes more error-prone with the newly added parameter.

Reply:
the command is passed like this: ``secureboot-certs install --user-agent="My user agent"`